doc/ModuleCreation.md

```diff
@@ -1,198 +1,40 @@
 ## Creating New Modules
-### General Nodes
-For a "Hello World" example see [here](https://github.com/mrkno/HelloKassy).
-
-The basic module for a Kassy module will contain the following files:
-- `someModule.js` - a javascript file that will be the basis of the module.
-- `kassy.json` - a file describing your module, that can be used by Kassy to start it.
-
-Correctly created modules once in a public git repository can automatically be installed using the `kpm` command.
-
-#### File Locations
-
-- Modules should be created as their folder within the `modules` (<b>not `node_modules`</b>) subdirectory.
-- Each module folder should contain a `kassy.json` file (see existing modules for examples).
-
-#### `require('module')`
-Any modules that depend on an `npm` package should include it as usual using the `require` function. Note: if the package that has been required is not installed, the latest version will be installed for you.
-
-Additional methods are also avalible with require:
-
-* ~~require.safe('module')~~ __Deprecated__ available for backwards compatibility.
-* ~~require.once('module')~~ __Internal__ required to allow seemless code hotswap.
+### Quick Reference
+- [Api Object](./api/Api.md)
+- [Event Object](./api/Event.md)
+- [`kassy.json`](./api/Kassy.json.md)
+- [Module Methods](./api/Module.md)
+- [Translations](./api/Translation.md)
+
+### Module Formats
+Concierge supports two primary module formats:
+- Kassy (see below for details, recommended). This is the native module format for Concierge and is the most supported.
+- Hubot. For details on how to create Hubot scripts, refer to the Hubot documentation [here](https://hubot.github.com/docs/scripting/).
+
+### Kassy Module Format
+For a "Hello World" example see [here](https://github.com/concierge/HelloConcierge).
+
+The basic module for a Concierge module will created as a new directory within the `modules` directory of Concierge (*not to be confused with `node_modules`</b>*).
+It will contain the following files:
+- `someModule.js` - an ES6, strict mode JavaScript file that will be the basis of the module. [See here](./api/Module.md) for the methods that this should implement.
+- `kassy.json` - a file describing your module, that can be used by Concierge to start it. [See here](./api/Kassy.json.md) for the properties that should be in this file.
+
+#### Dependencies
+Concierge has been created using [Node.JS](https://nodejs.org/). As with any Node.JS application, it is possible to depend on other Node.JS modules using `require`. Within Concierge, `require` has been extended to automatically install any dependency it does not have/cannot find from [NPM](https://www.npmjs.com/).
```

Comment:
Ambiguious sentence 

> any dependency it does not have/cannot find from NPM

Implies modules that were not found on NPM.

Can be replaced with 

> Within Concierge, `require` has been extended to automatically install modules from NPM if they are not found locally.